website/servers/index.php

```diff
@@ -117,9 +129,11 @@ function cmpByUsercount($b, $a) {
 				<div class="formrow">
 					<label class="label">Owner's username: <input class="textbox" type="text" name="owner" placeholder="(Optional)" /><em>(separate multiple owners by commas)</em></label>
 				</div>
+				<div class="formrow">
+					<label class="label">Owner's email: <input class="textbox" type="text" name="email" /></label>
 				<div class="buttonrow">
 					<button type="submit"><strong>Add server</strong></button>
-				</div>
+				</div></div>
```

Comment:
```suggestion
				</div>
```

Why are you adding the `</div>` here? The indentation is wrong for this, anyway.